index.js

```diff
@@ -28,6 +29,7 @@ const docSchema = {
   links: { type: 'TEXT', notnull: 1, dflt_value: null, pk: 0 },
   forks: { type: 'TEXT', notnull: 1, dflt_value: null, pk: 0 },
   updatedAt: { type: 'TEXT', notnull: 1, pk: 0 },
+  deleted: { type: 'INTEGER', notnull: 0, pk: 0 },
```

Comment:
Not sure if it makes sense to make it notnull? Either way you will have to decide what to do with the `null` value returned on read.

Reply:
Ok, so despite `deleted` being optional and always `false` in the schema. In the table is required and initialized to false, right?